server/rpm/pbench-server.spec.j2

```diff
@@ -65,6 +65,9 @@ cd /%{installdir}
 rm -rf node_modules
 npm install
 
+# install selinux modules under /%{installdir}
+dnf install python3-libselinux
```

Comment:
Actually, since we are in an RPM `.spec` file, we can add a `Requires: python3-libselinux` line to replace line 21.

Reply:
ohh ok.. sorry I was not sure how I can add